app/src/components/ErrorTextBox.tsx

```diff
@@ -0,0 +1,51 @@
+import { useTheme } from '@hyperledger/aries-bifold-core'
```

Comment:
I think we have a component called InfoBox that takes in an option to be Info, Warning, or Error. Can we-reuse that component? I think it appears at the top of the Terms screen.

Reply:
There is a component called InfoBox which is configurable but it is made for modal use and doesn't work for this use case. There is another component from Bifold called InfoTextBox which is a bit similar but it is not configurable and doesn't have quite the same positioning style as in the wireframes.